polynote-kernel/src/main/scala/polynote/kernel/remote/transport.scala

```diff
@@ -338,22 +339,33 @@ object SocketTransport {
 
     private def listJars(path: Path): RIO[BaseEnv, Seq[Path]] = listFiles(path)
       .map(_.filter(_.getFileName.toString.endsWith(".jar")))
-      .catchSome {
-        case NonFatal(err) => Logging.error(s"Failed to list JARs in $path", err).as(Seq.empty)
+      .tapError {
```

Comment:
Why the change from `catchSome` -> `tapError` ? With this change the error will bubble up, right?

Reply:
Yes, I want it to bubble up so I can fall back to inheriting the classpath of the server (so things can run from IDE or sbt without having to build the distribution first)

Reply:
Ah, so where is the error being caught now?